### PR TITLE
[#111] Smart Examples in Swagger

### LINKED ISSUE #111: Smart Examples in Swagger
Implement smart swagger examples handling - for now their code is commented. Problematic part is using right DB and checking if this DB has needed records.

### PR DESCRIPTION
**Pull request by JanKapala**
Fixed #111

### CODE REVIEW

**Review by wujuu — APPROVED** [review]
Awesome

File: recommender/utils.py [hunk 1]
```diff
@@ -154,3 +156,40 @@ def gen_json_dict(panel_id: str, anonymous_user: bool = False) -> Dict[str, Any]
...
+    examples = {
+        "categories": [1, 2, 3],
+        "geographical_availabilities": [1, 2, 3],
+        "providers": [1, 2, 3],
+        "related_platforms": [1, 2, 3],
+        "scientific_domains": [1, 2, 3],
+        "target_users": [1, 2, 3],
+    }
+
+    if os.environ["FLASK_ENV"] == "testing":
```
Comment (wujuu) [inline recommender/utils.py:180]: I would add a comment on why we don't use current_app.config - it's a special case

Comment (JanKapala) [reply recommender/utils.py:180]: bo tego tam nie ma? XD

Comment (wujuu) [reply recommender/utils.py:180]: nie

Comment (JanKapala) [reply recommender/utils.py:180]: no nie ma

**Review by wujuu — APPROVED** [review]

**Pull request merged**